kimchi/src/verifier_index.rs

```diff
@@ -64,7 +66,7 @@ pub struct VerifierIndex<G: CommitmentCurve> {
     pub max_quot_size: usize,
     /// polynomial commitment keys
     #[serde(skip)]
-    pub srs: Arc<SRS<G>>,
+    srs: OnceCell<Arc<SRS<G>>>,
```

Comment:
This field (and the others below) need to be public, otherwise the mina side won't be able to create a verifier index from one it receives over the network.

Reply:
my bad on this. not aware how mina does things